{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "10629322",
  "multipleFaultNumbers": null,
  "date": "2020-12-07T16:48:17",
  "region": "Ashton-Under-Lyne",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-07T16:48:17",
  "information": "",
  "actualTimeOfRestoration": "2020-12-07T16:48:17",
  "outageCentrePoint": {
    "lat": 53.499748,
    "lng": -2.062971
  },
  "outageLocations": null,
  "AffectedPostcodes": " OL6 9AX, OL6 9BG, OL6 9BY, OL6 9DF, OL6 9DG, OL6 9DH, OL6 9DN, OL6 9DW",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GS"
}